src/diceBot/CodeLayerd.rb

```diff
@@ -31,48 +40,57 @@ def rollDiceCommand(command)
     result = ''
 
     case command
-    when /(\d+)?CL(\@?(\d))?(>=(\d+))?/i
-      base = (Regexp.last_match(1) || 1).to_i
-      target = (Regexp.last_match(3) || 6).to_i
-      diff = Regexp.last_match(5).to_i
-      result = checkRoll(base, target, diff)
+    when /(\d+)?CL([+-]\d+)?(\@(\d))?(\[(\d{1,2})\])?([+-]\d+)?(>=(\d+))?/i
```

Comment:
```suggestion
    when /(\d+)?CL([+-]\d+)?(\@(\d))?(\[(\d+)\])?([+-]\d+)?(>=(\d+))?/i
```
`\d{1,2}` は若干わかりづらいので、 `\d+` にしてしまって問題ないと思います